tests/utils.py

```diff
@@ -1143,6 +1137,9 @@ def assertBinaryOperation(self, x_values, y_values, operation, format, substitut
         vars()['test_ne_%s' % datatype] = _binary_test(
             'test_ne_%s' % datatype, 'x != y', examples
         )
+        vars()['test_matmul_%s' % datatype] = _binary_test(
+            'test_matmul_%s' % datatype, 'x * y', examples
```

Comment:
This and the inplace one below should use @ not *.

Reply:
Updated to use small ints :+1: